src/components/Alert/Alert.js

```diff
@@ -1,189 +1,110 @@
-/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable object-curly-newline */
```

Comment:
@josecarlosgonzalezv Need this as it is the only conflicting error between prettier and eslint setting when we run npm run check